### PR TITLE
RF-1.0.1.1.1 Updated script for help

### PR DESCRIPTION
**Pull request by ananashraf**
- Syntax for commands corrected.
- Update and selfupdate commands added.

### CODE REVIEW

**Comment by asa1997** [comment]
@panickervinod This is our new contributor.

**Comment by asa1997** [comment]
@asa1997 please validate with review checklist & test case.

File: src/main/bash/commands/kobman-help.sh [hunk 1]
```diff
@@ -4,51 +4,64 @@ function __kob_help {
 
 	__kobman_echo_yellow "KOBman - Help"
 	__kobman_echo_blue "------------------------------------------------------------"
```
Comment (panickervinod) [inline src/main/bash/commands/kobman-help.sh:6]: @asa1997  please refactor, avoid multiple echo function call.

File: src/main/bash/commands/kobman-help.sh [hunk 2]
```diff
@@ -4,51 +4,64 @@ function __kob_help {
...
+	__kobman_echo_white "<qualifier>  : environment, version, namespace"
+	__kobman_echo_white ""
+	__kobman_echo_white "environment"
 	__kobman_echo_blue "-------------------------------------------------------------"
-	__kobman_echo_white "A Namespace is a mechanism to partition resources created by"
-	__kobman_echo_white "users into a logically named group."
 	__kobman_echo_white ""
-	__kobman_echo_white "bcgov  		: The namespace of the British Columbia"
-	__kobman_echo_white "hyperledgerkochi: The namespace of Kochi"
+	__kobman_echo_white "von-network  : https://github.com/hyperledgerkochi/von-network"
```
Comment (panickervinod) [inline src/main/bash/commands/kobman-help.sh:14]: Environment information need not be in help as this is not static data.

**Comment by asa1997** [comment]
> @asa1997 please validate with review checklist & test case.
The test script will be submitted asap

**Pull request closed without merging**